styles/utility.css

```diff
@@ -7,14 +7,34 @@
     justify-content: center;
 }
 
+.justify-cont-end{
+    justify-content: end;
+}
+
+.justify-cont-spacebetween{
+    justify-content: space-between;
+}
+
 .align-item-center{
     align-items: center;
 }
 
+.flex-d-col{
+    flex-direction: column;
+}
+
+.flex-gap{
+    gap: 1rem;
+}
+
 .mr-small{
     margin-right: 0.5rem;
```

Comment:
I guess we should use px for anything less than 1 rem

Reply:
okay, i change that